shell.nix

```diff
@@ -0,0 +1,65 @@
+{ nixroot  ? (import <nixpkgs> {}) }:
+let inherit (nixroot) stdenv
+    chromaprint fftw flac libid3tag libmad libopus libshout libsndfile
+    libusb1 libvorbis libebur128 pkgconfig portaudio portmidi protobuf qt4 qt5
```

Comment:
I don't think both Qt4 and Qt5 should be required. How about just requiring Qt4 till we finally switch to Qt5? Or can you make Qt5 an optional dependency somehow?

Reply:
I put both in so you can choose which one you wan't to compile against and you can test both. I had problems getting scons to comply without having both installed. have to look into that some other time :)

Reply:
How about removing the Qt4 dependency and we can merge this PR when we switch to Qt5?

Reply:
We've switched to Qt5 now, so I think we can merge this if you remove the Qt4 dependency.